js/app/Visualization/Animation/AnimationManager.js

```diff
@@ -340,11 +340,11 @@ function(Class,
 
       if (rowidx) {
         var animation = self.animations[rowidx[0]];
-        if (animation.data_view) {
+        if (animation && animation.data_view) {
```

Comment:
This should not be possible. If rowidx is not null/undefined, rowidx[0] should be an animation id, or there is a bug somewhere else.

This is not a major issue, but I think that throwing an exception if animation is undefined is correct.